examples/simple_hpc_pbs_zfs/readme.md

```diff
@@ -0,0 +1,66 @@
+# Build a PBS compute cluster
+![Build Status](https://azurecat.visualstudio.com/hpccat/_apis/build/status/azhpc/examples/simple_hpc_pbs?branchName=master)

```

Comment:
This is a wrong link, it should be simple_hpc_pbs_zfs

Reply:
fixed

Reply:
done